### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by Rensvandeschoot**
This PR updates the readme file:

- removing ASReview specific information 
- add information about the MIT license

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ The datasets are alphabetically ordered. See [index.csv](index.csv) for all avai
 
 
 ## Publishing your data
-For publishing either your data and / or your AI-aided systematic review, we recommend using the Open Science frame (OSF). OSF is part of the Center for Open Science (COS), which aims at increasing openness, integrity, and reproducibility of research ([OSF](https://www.cos.io/our-products/osf), 2020). How to share your data using OSF: A [step-by-step guide](https://journals.sagepub.com/doi/pdf/10.1177/2515245918757689).
+For publishing either your data we recommend using the Open Science frame (OSF). OSF is part of the Center for Open Science (COS), which aims at increasing openness, integrity, and reproducibility of research ([OSF](https://www.cos.io/our-products/osf), 2020). How to share your data using OSF: A [step-by-step guide](https://journals.sagepub.com/doi/pdf/10.1177/2515245918757689).
```
Comment (J535D165) [inline README.md:50]:
```suggestion
For publishing either your data, we recommend using the Open Science frame (OSF). OSF is part of the Center for Open Science (COS), which aims at increasing openness, integrity, and reproducibility of research ([OSF](https://www.cos.io/our-products/osf), 2020). How to share your data using OSF: A [step-by-step guide](https://journals.sagepub.com/doi/pdf/10.1177/2515245918757689).
```

File: README.md [hunk 2]
```diff
@@ -60,26 +60,13 @@ When sharing your dataset or a link to your already published systematic review,
...
-
-### Examples
-Two examples of authors who have published their systematic review data online:
-- A systematic review on treatment for Wilson disease, in RIS format https://zenodo.org/record/3625931#.XvB\_92ozblw
-- Data from four systematic reviews on fault prediction in software engineering, in .csv format https://zenodo.org/record/1162952#.XvCCZmozblw.
+datasets. In each of these subfolders, the `.ipynb` script retrieve a dataset from OSF or Zenodo, and preprocesses it by adding customized labels and marking duplicates. The script also reports the inclusion rate, and missing patterns and word clouds of titles and abstracts. After preprocessing, an ASReview-ready dataset in `.csv` format is generated in the `output/` folder. Extensions .csv, .xlsx, and .xls. CSV files should be comma separated and UTF-8 encoded. To indicate labelling decisions, one can use "included" or "label\_included". This label should be filled with all 0’s and 1’s, where 0 means that the record is not included and 1 means included.
```
Comment (J535D165) [inline README.md:66]:
```suggestion
datasets. In each of these subfolders, the `.ipynb` script retrieves a dataset from OSF or Zenodo, and preprocesses it by adding customized labels and marking duplicates. The script also reports the inclusion rate, and missing patterns and word clouds of titles and abstracts. After preprocessing, an ASReview-compatible dataset in `.csv` format is generated in the `output/` folder. Extensions .csv, .xlsx, and .xls. CSV files should be comma-separated and UTF-8 encoded. To indicate labeling decisions, one can use "included" or "label_included". This label should be filled with all 0’s and 1’s, where 0 means that the record is not included and 1 means included.
```

File: README.md [hunk 3]
```diff
@@ -60,26 +60,13 @@ When sharing your dataset or a link to your already published systematic review,
...
 
+## License
+The scripts in the current project are MIT licensed. The datasets should have their own license allowing for re-use.
```
Comment (J535D165) [inline README.md:69]:
```suggestion
The scripts in the current project are MIT licensed. The datasets (should) have a permissive license.
```

**Review by J535D165 — CHANGES_REQUESTED** [review]
Thanks, fixed some typos. Didn't look too much into the content. I'm fine with merging.

**Comment by Rensvandeschoot** [comment]
thanks for the fast review!

**Pull request merged**